kalitectl.py

```diff
@@ -571,6 +572,27 @@ def start(debug=False, watch=False, daemonize=True, args=[], skip_job_scheduler=
             'server.socket_port': port
         })
 
+    # idealy we'd have a setting element, but these are imported much later
+    # and it will require a much deeper change
+    idle_timeout = int(os.getenv("KALITE_IDLE_TIMEOUT", 0))
```

Comment:
Even if 0 evaluates to false, I don't think is much more complicated, and it's definitely more clear and readable IMHO. I still think it makes sense